test/languages.ts

```diff
@@ -692,7 +692,6 @@ export const SwiftLanguage: Language = {
         { "access-level": "internal" },
         { "access-level": "public" },
         { protocol: "equatable" },
-        { "optional-enums": "true" },
```

Comment:
We removed this because optional-enums now requires an external dependency. Since the swift tests are built using ´swiftc´ it's hard to have dependencies. One option would be to switch to using spm and ´swift´ to build and install dependencies, but that would probably slow down the testing. Another would be to change the `import OptionallyDecodable` to be within `#if canImport` declarations and inline a OptionallyDecodable implementation in main.swift, but that seems like an ugly solution. For now we disabled the tests (in the same way it is disabled for Alamofire).